packages/app/Root.js

```diff
@@ -33,56 +34,52 @@ function TabNavigation() {
 
 function App() {
   const { user, setUser } = useUserContext();
+  const [isFirstTime, setIsFirstTime] = useState(null);
 
   const currentSession = async () => {
-    // automatically log in depending on which token is set in expo secure store
-
-    // log in with local auth
-    const localAuthToken = await SecureStore.getValueFor("local_auth_token");
-    if (localAuthToken) {
-      const payload = await getUserInfo(localAuthToken);
+    const token = await AsyncStorage.getItem('token');
+    const isFirstTime = await AsyncStorage.getItem('isFirstTime');
```

Comment:
Should isFirstTime be stored in AsyncStorage? I'm guessing it should be stored on a database somewhere so that the user only needs to do onboarding once. With AsyncStorage, if the user were to log on to another device, they would see the onboarding screen again.